src/dbus_api.rs

```diff
@@ -917,11 +917,12 @@ fn create_pool(m: &MethodInfo<MTFn<TData>, TData>) -> MethodResult {
     let name: &str = try!(get_next_arg(&mut iter, 0));
     let raid_level: u16 = try!(get_next_arg(&mut iter, 1));
     let devs: Array<&str, _> = try!(get_next_arg(&mut iter, 2));
+    let force: bool = try!(get_next_arg(&mut iter, 3));
```

Comment:
One last thought: You will also need to change the ```in_arg``` value for this method. That's the value that controls the information returned by the Introspect method.